underfs/hdfs/src/main/java/alluxio/UfsConstants.java

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
I think I overlooked this file since it just popped up on my staging area...

Reply:
I don't think this file is checked in?

Reply:
This is from multi-hdfs feature. It should be generated from the template. Maybe we need to add it to .gitignore?

Reply:
updated .gitignore